src/lib/app/templates.ts

```diff
@@ -85,7 +85,14 @@ export const defaultStandardCommunityRoles: RoleTemplate[] = [
 	{
 		name: 'Member',
 		default: true,
-		policies: [{permission: 'InviteToCommunity'}],
+		//TODO how to do this without raw strings & instead use permissions?
+		policies: [
+			{permission: 'InviteToCommunity'},
+			{permission: 'CreateSpace'},
+			{permission: 'ReadSpace'},
+			{permission: 'ReadSpaces'},
+			{permission: 'UpdateSpace'},
```

Comment:
```suggestion
			{permission: permissions.InviteToCommunity},
			{permission: permissions.CreateSpace},
			{permission: permissions.ReadSpace},
			{permission: permissions.ReadSpaces},
			{permission: permissions.UpdateSpace},
```

see the other comments -- also you'll need to add `permissions` to the imports